app/_how-tos/kic-proxy-weights.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Weight Traffic to Specific Backends
+description: "Distribute traffic across multiple Kubernetes Services in a single HTTPRoute"
+content_type: how_to
+
+permalink: /kubernetes-ingress-controller/routing/weights/
+breadcrumbs:
+  - /kubernetes-ingress-controller/
+  - index: kubernetes-ingress-controller
+    section: Routing
+
+products:
+  - kic
+
+works_on:
+  - on-prem
+  - konnect
+
+entities: []
+
+tldr:
+  q: How do I route HTTP traffic with a custom weight per backend using {{ site.kic_product_name }}?
+  a: Create a `HTTPRoute` resource, and specify a `weight` property under `spec.rules[*].backendRefs[*].weight`
```

Comment:
```suggestion
  a: Create a `HTTPRoute` resource, and specify a `weight` property under `spec.rules[*].backendRefs[*].weight` to route traffic to specific backends.
```
Currently we don't show the `q` of the `tldr` on the page, so this is out context. Maybe we should.